src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -465,6 +466,84 @@ private static string GetParameterText(string parameterName)
             }
         }
 
+        internal static void InvokePipelineInBackground(
+                                            CommandBaseAst[] pipeElementAsts,
```

Comment:
You can pass the `PipelineAst` instead of the elements because that's all you use here.